introduction_to_applying_machine_learning/visual_object_detection/visual_object_detection.ipynb

```diff
@@ -0,0 +1,2889 @@
+{
```

Comment:
<p>Clear all outputs - this is because the outputs are different each time and will make future changes to the notebook easier</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>